### PR TITLE
MNT getSummaryThumbnail is deprecated

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-elemental-fileblock/issues/39

### CODE REVIEW

File: src/Block/FileBlock.php [hunk 1]
```diff
@@ -74,6 +75,10 @@ protected function provideBlockSchema()
      */
```
Comment (ChloeSartorelli) [inline src/Block/FileBlock.php:76]:
```suggestion
     * @deprecated 3.1.0 Will be removed without equivalent functionality to replace it
     */
```

File: src/Block/FileBlock.php [hunk 2]
```diff
@@ -74,6 +75,10 @@ protected function provideBlockSchema()
...
     public function getSummaryThumbnail()
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('1.13.0', 'Will be removed without equivalent functionality to replace it.');
+        });
```
Comment (ChloeSartorelli) [inline src/Block/FileBlock.php]:
```suggestion
        Deprecation::notice('3.1.0', 'Will be removed without equivalent functionality to replace it.');
```
Fix version number.
Also don't wrap it in `withNoReplacement()` - that's only used for something we're calling from internal code, which isn't the case here. Wrapping with this method would actually _suppress_ the deprecation warning, which we don't want to do for this method.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/Block/FileBlock.php (hunk 1 above)
Comment (sabina-talipova) [reply src/Block/FileBlock.php:76]: Updated

File: src/Block/FileBlock.php (hunk 2 above)
Comment (sabina-talipova) [reply src/Block/FileBlock.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**